tasks/main.yml

```diff
@@ -2,6 +2,27 @@
 - name: Set version specific variables
   include_tasks: set_vars.yml
 
+- name: Scan sudoers
+  scan_sudoers:
+    output_parsed_configs: True
+    output_raw_configs: False
```

Comment:
```suggestion
    output_raw_configs: false
```

Reply:
Are there any passwords, keys, or other secrets in the sudoers file?  Because this will likely leak the contents to the logs.  If it could leak secrets, add a `no_log: true` to the task.

Reply:
I added it there, it seems to be applicable.